sphinx/templates/quickstart/conf.py_t

```diff
@@ -132,6 +132,11 @@ latex_elements = {
     # Latex figure (float) alignment
     #
     # 'figure_align': 'htbp',
+
+    # Chinese language default package ctex
```

Comment:
Why is this needed? ctex is already loaded by default settings (see `default_latex_elements()`).